File: 백준/Bronze/2562. 최댓값/최댓값.py
import sys
input = sys.stdin.readline

num = [int(input()) for _ in range(9)]
result = num[0]
index = 1

for i in range(len(num)):
    if result < num[i]:
        result = num[i]
        index = i+1

print(result)
print(index)

File: 백준/Bronze/2562. 최댓값/최댓값.js
const input = require('fs').readFileSync(process.platform === 'linux' ? '/dev/stdin' : 'input.txt').toString().trim().split('\n');

const nums = input.map(Number);
const max = Math.max(...nums)
let count = 0;
for (num of nums){
    count++;
    if(max == num) break;
}
console.log(max + '\n' + count)